CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [unreleased]
```

Comment:
- Update the patch version please.
- Is there anything we can add to the tests / CICD / pre commit hooks that can ensure this type of error doesn’t happen again?
- One of the PR checks for the example app is failing. Please see why.
- You can remove the example pr checks for emailpassword, passwordless, thirdparty, thirdpartyemailpassword, thirdpartypasswordless and supabase.